allennlp/run.py

```diff
@@ -0,0 +1,128 @@
+from typing import Any, Dict, Union
```

Comment:
I have never made a pip installable package. let me do some research here. is our goal to make allennlp pip-installable? I also prefer having a single command line tool with options e.g. `allennlp train` and `allennlp eval` but maybe I'm getting too far ahead